lib/matplotlib/axis.py

```diff
@@ -1154,6 +1162,11 @@ def set_default_intervals(self):
         # attribute, and the derived code below will check for that
         # and use it if it's available (else just use 0..1)
 
+    def _get_axis_name(self):
```

Comment:
could probably be higher in the file (e.g. next to _get_shared_axis/axes; perhaps all three could go just after init or after the various isDefault_foos.

Reply:
ping @oscargus here?

Reply:
I've been off for a bit, but can for sure move it in the near future!